utils/src/main/java/uk/gov/pay/commons/utils/logging/LoggingFilter.java

```diff
@@ -16,6 +17,18 @@
 public class LoggingFilter implements Filter {
 
     private static final Logger logger = LoggerFactory.getLogger(LoggingFilter.class);
+    // This key should match the value in our logging configuration e.g. %X{X-Request-Id:-(none)}
```

Comment:
If we use a Javadoc comment here then IDEs can show display the note wherever the field is used

```java
/**
 * This key should match the value in our logging configuration e.g. %X{X-Request-Id:-(none)}
 */
private static final String MDC_REQUEST_ID_KEY = "X-Request-Id";
```